source/beamline/PGM/PGMBeamline.cpp

```diff
@@ -147,6 +169,18 @@ void PGMBeamline::setupExposedDetectors()
 	addExposedDetector(i0EndstationBladeCurrentDetector_);
 	addExposedDetector(i0BeamlineBladeCurrentDetector_);
 	addExposedDetector(photodiodeBladeCurrentDetector_);
+
+	addExposedDetector(oceanOpticsDetector_);
+}
+
+bool PGMBeamline::getConnectedState() const
```

Comment:
in another PR of mine, this function is called `isConnected()` and I think that is a common name we used for the other components.